torch/_library/utils.py

```diff
@@ -156,3 +157,67 @@ def zip_schema(schema, args, kwargs):
             continue
         yield info, args[i]
     return
+
+
+def mangle(value: str) -> str:
+    """Mangles a string into a usable Python identifier.
+
+    https://docs.python.org/3/reference/lexical_analysis.html#identifiers
```

Comment:
I'm surprised there is no generic tool to make any string into a indentifier...

Reply:
This doesn't handle if a value is a keyword, right? That can be detect with lib/keyword.py

Reply:
There is does github gist that seems to cover some more edgecases https://gist.github.com/JamesPHoughton/3a3f87c6662bf5c9eccc9f2206e228fd